src/cargo/core/features.rs

```diff
@@ -892,19 +954,19 @@ fn parse_git(it: impl Iterator<Item = impl AsRef<str>>) -> CargoResult<Option<Gi
 
     for e in it {
         match e.as_ref() {
+            "all" => return Ok(Some(GitFeatures::all())),
```

Comment:
I don't think we need to support string `"all"`. `-Zgit` or `--config unstable.git=true` should be sufficient?